debian/control

```diff
@@ -0,0 +1,22 @@
+Source: truenas-api-client
+Section: contrib/python
+Priority: optional
+Maintainer: Vladimir Vinogradenko <vladimirv@ixsystems.com>
+Build-Depends: debhelper-compat (= 12),
+               dh-python,
+               python3-all,
+               python3-fastentrypoints,
+               python3-libzfs,
```

Comment:
I don't like making our websocket client dependent on this package. I'd rather just hard-code these ZFS errors inside this module itself and deal with maintaining it.